db/migrations/20210224083353_create_dog_bark.sql

```diff
@@ -0,0 +1,32 @@
+-- +goose Up
+CREATE TABLE maker.dog_bark
+(
+    id         SERIAL PRIMARY KEY,
+    log_id     BIGINT  NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
+    header_id  INTEGER NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
+    address_id BIGINT  NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
+    ilk_id     INTEGER NOT NULL REFERENCES maker.ilks (id) ON DELETE CASCADE,
+    urn_id     INTEGER NOT NULL REFERENCES maker.urns (id) ON DELETE CASCADE,
+    clip       BIGINT  NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
```

Comment:
Was going back and forth on whether we wanted this to be a FK to the address table, or if we'd want to persist the clip address as a string. Curious what others think!

Reply:
I think its more useful to see the actual address as a user, unless we're missing something else in postgraphile by not having it be a FK.